api_tests/users/views/test_user_draft_registration_list.py

```diff
@@ -0,0 +1,131 @@
+import pytest
+from django.utils import timezone
+
+from api.base.settings.defaults import API_BASE
+from api.users.views import UserDraftRegistrations
+from api_tests.nodes.views.test_node_draft_registration_list import DraftRegistrationTestCase
+from api_tests.utils import only_supports_methods
+from osf.models import RegistrationSchema
+from osf_tests.factories import (
+    RegistrationFactory,
+    AuthUserFactory,
+    DraftRegistrationFactory,
+)
+from osf.utils import permissions
+from website.project.metadata.schemas import LATEST_SCHEMA_VERSION
+
+
+@pytest.mark.django_db
+class TestDraftRegistrationList(DraftRegistrationTestCase):
+
+    @pytest.fixture()
+    def other_admin(self, project_public):
+        user = AuthUserFactory()
+        project_public.add_contributor(user, permissions=[permissions.ADMIN], save=True)
+        return user
+
+    @pytest.fixture()
+    def schema(self):
+        return RegistrationSchema.objects.get(
+            name='Open-Ended Registration',
+            schema_version=LATEST_SCHEMA_VERSION)
+
+    @pytest.fixture()
+    def draft_registration(self, user, project_public, schema):
+        return DraftRegistrationFactory(
+            initiator=user,
+            registration_schema=schema,
+            branched_from=project_public
+        )
+
+    @pytest.fixture()
+    def url_draft_registrations(self, project_public):
+        return '/{}users/me/draft_registrations/'.format(API_BASE)
```

Comment:
I don't remember how /me/ works internally, but I assume this should work if using my own guid. Is that worth adding a test for? Overkill for all functionality, but a single get might be nice.

Reply:
Added in https://github.com/CenterForOpenScience/osf.io/pull/8681/commits/8ae8831c237feb0d44cd2107c91425b156fa6136#diff-024e783d133d520975d82387da8f4e23R133